assets/js/autocomplete/suggestions.js

```diff
@@ -133,12 +145,20 @@ function childNodeSuggestion (childNode, parentId, query, category, label) {
 function nodeSectionSuggestion (node, section, query, category, label) {
   if (!matchesAny(section.id, query)) { return null }
 
+  let link
+
+  if (section.anchor === '' || !section.anchor) {
```

Comment:
This is required because when using `searchData` there may be nodes with no anchor link.

Reply:
Is the `.anchor` never not set? It seems before it assumed it to always be present.

Reply:
Right, the second is unnecessary